src/foundry/client/data/documents/fog-exploration.d.mts

```diff
@@ -208,9 +208,9 @@ declare global {
 
       /**
        * The timestamp at which this fog exploration was last updated
-       * @defaultValue `Date.now()`
+       * @defaultValue `Date.now`
```

Comment:
So `initial` is set to `Date.now` but functions are called, therefore it was accurate to say that the default value was `Date.now()`.